packages/camera/camera_android_camerax/lib/src/camera_selector.dart

```diff
@@ -0,0 +1,153 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+
+import 'camera_info.dart';
+import 'camerax.pigeon.dart';
+import 'instance_manager.dart';
+import 'java_object.dart';
+
+/// Selects a camera for use.
```

Comment:
For this and the other native classes, could you link to the android docs. e.g. add

```
/// See https://developer.android.com/reference/androidx/camera/core/CameraSelector.
```